### PR TITLE
i18n: guard `ngI18nClosureMode`

### PR DESCRIPTION
**Pull request by petebacondarwin**
cc @filipesilva and @mhevery

### CODE REVIEW

File: packages/core/src/application_module.ts [hunk 1]
```diff
@@ -49,7 +49,8 @@ export function _localeFactory(locale?: string): string {
  * * Ivy enabled: use `$localize.locale`
  */
 export function getGlobalLocale(): string {
-  if (ngI18nClosureMode && typeof goog !== 'undefined' && goog.LOCALE !== 'en') {
+  if (typeof ngI18nClosureMode === 'undefined' && ngI18nClosureMode &&
```
Comment (filipesilva) [inline packages/core/src/application_module.ts]: Shouldn't this be `typeof ngI18nClosureMode !== 'undefined'` (`===` -> `!==`)?

**Review by filipesilva — APPROVED** [review]
LGTM aside from the one comment.

File: packages/core/src/application_module.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/core/src/application_module.ts]: !! Yes !! Doh.

**Comment by mhevery** [comment]
[presubmit](http://test/OCL:283572159:BASE:283572210:1575397546661:3a955d2f)

**Pull request closed without merging**